libbpf-tools/uprobe_helpers.c

```diff
@@ -261,6 +266,24 @@ off_t get_elf_func_offset(const char *path, const char *func)
 	}
 
 out:
+	if (ehdr.e_type == ET_EXEC || ehdr.e_type == ET_DYN) {
+		if (elf_getphdrnum(e, &nhdrs) != 0) {
+			ret = -1;
+			goto errout;
+		}
+		for (i = 0; i < (int)nhdrs; i++) {
+			if (!gelf_getphdr(e, i, &phdr))
+				continue;
+			if (phdr.p_type != PT_LOAD || !(phdr.p_flags & PF_X))
+				continue;
+			if (phdr.p_vaddr <= ret && ret < (phdr.p_vaddr + phdr.p_memsz)) {
+				ret = ret - phdr.p_vaddr + phdr.p_offset;
+				goto errout;
+			}
+		}
+		ret = -1;
+	}
+errout:
```

Comment:
Since this is used for a non-error condition on line 281, could you rename it to something without `err` in the name, or add a second label so that 281's `goto` is easily understood to not be an error state?

Reply:
Done.